### PR TITLE
Bug fixes and suggestion

### PR DESCRIPTION
**Pull request by validol731**
+Fix increasing hp "Command add".
+Fix decreasing hp "Command remove".
+Fix setting hp "Command set".
+Fix UI message about increasing hp.
+Fix UI message about decreasing hp.
+Fix UI message about setting hp.
+Fix UI message about decreasing hp after death.
+Fix increasing hp after killing mobs.
+Fix UI message about increasing hp after killing mobs.
+Added config simple configuration vanilla mobs to regen health.
+Added UI message about yet to killing N mob to regen X health

### CODE REVIEW

File: src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 			minimumHealth = builder
 					.comment("Minimum amount of health the player can end up with (2 = 1 heart) [default: 2]")
-					.defineInRange("minimumHealth", 2, 1, Integer.MAX_VALUE);
+					.defineInRange("minimumHealth", 6, 1, Integer.MAX_VALUE);
```
Comment (Mrbysco) [inline src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java:34]: Why did you change the default config value?

File: src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java [hunk 2]
```diff
@@ -51,7 +51,7 @@
 
 			regenHealth = builder
 					.comment("When set to true allows you to gain back health upon killing set target(s) [default: false]")
-					.define("regenHealth", false);
+					.define("regenHealth", true);
```
Comment (Mrbysco) [inline src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java:54]: Why did you change the default config value?

File: src/main/java/com/mrbysco/unhealthydying/handlers/HealthHandler.java [hunk 3]
```diff
@@ -20,13 +24,15 @@ public void onJoin(PlayerLoggedInEvent event) {
 			UnhealthyHelper.syncHealth(player);
 		}
 	}
-
+	public static AttributeModifier getModifier(double modifier) {
```
Comment (Mrbysco) [inline src/main/java/com/mrbysco/unhealthydying/handlers/HealthHandler.java:27]: Why duplicate the `getModifier` from `UnhealthyHelper`?

File: src/main/java/com/mrbysco/unhealthydying/util/UnhealthyHelper.java [hunk 4]
```diff
@@ -30,11 +30,20 @@ public static void changeModifier(PlayerEntity player, double modifierValue) {
             if(attributeInstance != null) {
                 AttributeModifier oldAttribute = attributeInstance.getModifier(Reference.HEALTH_MODIFIER_ID);
                 if(oldAttribute != null) {
-                    if(oldAttribute.getAmount() != modifierValue)
-                        HealthUtil.sendHealthMessage(player, (int)attributeInstance.getValue(), (int)modifierValue);
-
+                    if(oldAttribute.getAmount() != modifierValue) {
+                        int newModifier = -(20 + (int) modifierValue);
```
Comment (Mrbysco) [inline src/main/java/com/mrbysco/unhealthydying/util/UnhealthyHelper.java:34]: Should be using `attributeInstance.getBaseValue()` in case an external mod changes the base value

File: src/main/java/com/mrbysco/unhealthydying/util/UnhealthyHelper.java [hunk 5]
```diff
@@ -30,11 +30,20 @@ public static void changeModifier(PlayerEntity player, double modifierValue) {
...
+                        if (oldAttribute.getAmount() < modifierValue)
+                            HealthUtil.sendHealthMessage(player, newModifier, 1);
+                        else
+                            HealthUtil.sendHealthMessage(player, newModifier, -1);
+                    }
                     attributeInstance.removePermanentModifier(Reference.HEALTH_MODIFIER_ID);
                 }
+                else
+                {
+                    int newModifier = -(20 + (int) modifierValue);
```
Comment (Mrbysco) [inline src/main/java/com/mrbysco/unhealthydying/util/UnhealthyHelper.java:44]: Should be using `attributeInstance.getBaseValue()` in case an external mod changes the base value

**Review by Mrbysco — COMMENTED** [review]
I have some questions

File: src/main/resources/assets/unhealthydying/lang/en_us.json [hunk 6]
```diff
@@ -26,6 +26,7 @@
     "unhealthydying:setmaxhealth.message": "§aYour max health has set to §r%s§a.§r",
     "unhealthydying:addhearts.message": "§aYour health has been increased by %s§r §aheart(s).",
     "unhealthydying:removehearts.message": "§cYour health has been reduced by %s §cheart(s).",
+    "unhealthydying:killingmobscount.message": "§aYou need yet to kill %s§r §a%s§r§a to receive %s§r §ahalf heart(s).",
```
Comment (Mrbysco) [inline src/main/resources/assets/unhealthydying/lang/en_us.json:29]: Should probably be "You have yet to kill ..."

File: src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java (hunk 1 above)
Comment (validol731) [reply src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java:34]: Sorry, you may remove these changes

File: src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java (hunk 2 above)
Comment (validol731) [reply src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java:54]: Sorry, you may remove these changes

**Comment by validol731** [comment]
I create a new pull
